### PR TITLE
Feature/contentful importer with flotiq cli

### PR DESCRIPTION
**Pull request by trzcina**

### CODE REVIEW

**Comment by WHLukasz** [comment]
New feature, that allows for data migration from Contentful space to Flotiq.
It imports 3 types od data: content types, assets and content objects.
The results of migration are displayed in three different tables. Each of them displays total success and error counts of the import and detailed information about the errors in the `Warnings` section.
importCtdSs
importMediaSs
importCoSs

File: src/contentful-import/notify.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            if (warnings[element][2] === "batchError") {
+                notifyTable.push([
+                    warnings[element][0],
+                    {
+                        colSpan: 2,
+                        content: typeOutWarnings(warnings[element][1], "General batch error:")
+                    }
+                ]);
+            } else {
+                for (object in warnings[element][1]) {
```
Comment (KarolNet) [inline src/contentful-import/notify.js:152]: add 'let'

File: src/contentful-import/notify.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                }
+            }
+        }
+    }
+    console.log(notifyTable.toString());
+}
+function typeOutWarnings(data, starting_line = "") {
+    let text = "";
+    text += starting_line;
+    for (field in data) {
```
Comment (KarolNet) [inline src/contentful-import/notify.js:167]: add 'let'

File: src/contentful-import/flotiq-contentful-import.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
+const config = require("../configuration/config");
+const contentfulExport = require('contentful-export');
```
Comment (KarolNet) [inline src/contentful-import/flotiq-contentful-import.js:2]: missing contentfull-export in packages.json dependencies

File: src/contentful-import/flotiq-contentful-import.js [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+const path = require('path');
+const cfHtmlRenderer = require('@contentful/rich-text-html-renderer/dist/rich-text-html-renderer.es5');
+const { resultNotify } = require('./notify');
+const { flotiqMedia, cfMediaToObject} = require('./media');
+const { flotiqCtdUpload, flotiqCoUpload, flotiqMediaUpload } = require('./upload');
+
+module.exports = contentful = async (flotiq_ApiKey, cont_spaceId, cont_contentManagementApiKey, translation = "en-US") => {
```
Comment (KarolNet) [inline src/contentful-import/flotiq-contentful-import.js:9]: use same name schemat like on project: flotiq_ApiKey -> flotiqApiKey itp.

File: src/command/command.js [hunk 5]
```diff
@@ -90,6 +90,24 @@ yargs
             wordpressStart(argv.flotiqApiKey, argv.wordpressUrl, yargs.argv['json-output']);
         }
     })
+    .command('contentful-import [flotiqApiKey] [cf_spaceId] [cf_contentManagementApiKey] [translation]', 'Import Contentful to Flotiq', (yargs) => {
```
Comment (KarolNet) [inline src/command/command.js:93]: Differents parameters between --help and readme, use full camelcase

File: src/command/command.js [hunk 6]
```diff
@@ -90,6 +90,24 @@ yargs
             wordpressStart(argv.flotiqApiKey, argv.wordpressUrl, yargs.argv['json-output']);
         }
     })
+    .command('contentful-import [flotiqApiKey] [contentfulSpaceId] [contentfulContentManagementToken] [translation]', 'Import Contentful to Flotiq', (yargs) => {
```
Comment (KarolNet) [inline src/command/command.js:93]: parameter [flotiqApiKey] is optional (can be read from ENV) move this to end, before [translation]

File: src/command/command.js [hunk 7]
```diff
@@ -90,6 +90,24 @@ yargs
...
+    }, async (argv) => {
+        const contentful = require('../contentful-import/flotiq-contentful-import.js');
+        if (yargs.argv._.length < 3 || yargs.argv._.length === 3 && !apiKeyDefinedInDotEnv()) {
+            const answers = await askQuestions(questionsText.CONTENTFUL_IMPORT);
+            let { flotiqApiKey, contentfulSpaceId, contentfulApiKey } = answers;
+            await contentful(flotiqApiKey, contentfulSpaceId, contentfulApiKey);
+        } else if (yargs.argv._.length === 3 && apiKeyDefinedInDotEnv()) {
+            contentful(process.env.FLOTIQ_API_KEY, argv.contentfulSpaceId, argv.contentfulContentManagementToken);
```
Comment (KarolNet) [inline src/command/command.js:101]: more 'await', always when function contentful is executed

**Pull request merged**